core/tests/test_utils.py

```diff
@@ -1307,12 +1239,9 @@ def set_interaction_for_state(self, state, interaction_id):
             interaction_id: str. The interaction id to set. Also sets the
                 default customization args for the given interaction id.
         """
-
         # We wrap next_content_id_index in a dict so that modifying it in the
         # inner function modifies the value.
-        next_content_id_index_dict = {
-            'value': state.next_content_id_index
-        }
+        next_content_id_index_dict = {'value': state.next_content_id_index}
```

Comment:
Generally I think we slightly prefer the extended notation for dicts (break after '{'), where each key-value pair is on a single line.

I don't feel too strongly, just mentioning.